client/components/blazepress-widget/style.scss

```diff
@@ -77,6 +80,54 @@ $headerHeight: 72px;
 			margin-top: 0;
 		}
 	}
+
+	&.blazepress-i2 {
+		.blazepress-widget__header-bar {
```

Comment:
I noticed that currently we have a bottom border and it looks like this:
<img width="1290" alt="image" src="https://github.com/Automattic/wp-calypso/assets/115007291/b1f5033d-1306-48a6-a6ff-7195e56fb4e3">

but there's no one in [Figma](https://www.figma.com/file/uvh2KtY65b7hGBHI9exbJ1/BlazePress?type=design&node-id=5999-48155&mode=design&t=qLYKjV1VailCScfp-0):
<img width="1019" alt="image" src="https://github.com/Automattic/wp-calypso/assets/115007291/64ce0232-c10a-40d6-ab59-b459ccde08eb">

Reply:
I created a ticket to change that behavior (SELFSERVE-655). I talked with Bart, and he wants the border-bottom to appear only after the user scrolls the page. 

I experimented a little to override the scroll with CSS, make calypso handle the scroll and add an event there so we could track the scroll, but the solution got a bit messy.

I thought of having this PR with the new design a little thinner and adding the scroll logic in the next one.